keccak/Cargo.toml

```diff
@@ -14,5 +14,9 @@ categories = ["cryptography", "no-std"]
 readme = "README.md"
 
 [features]
+asm = []       # Use optimized assembly when available (currently only ARMv8)
```

Comment:
Should this perhaps be a `cfg!` attribute rather than a crate feature?

Reply:
Hm, I am not sure. Application developers would probably want to have it enabled by default without any additional steps from users. It's a slightly different situation from the `aes` crate where configuration flags are used mostly for testing backends.